www/configurator/src/index.js

```diff
@@ -33,36 +33,63 @@ if (process.env.NODE_ENV === 'production') {
   cssPath = 'test-dist/uncompiled-css/templates/';
 }
 
+// Function to get params on the page
+function getUrlParams() {
+  const params = {
+    search: {},
+    hash: {}
+  };
+  const templateParam = queryString.parse(location.search.substring(1)).template;
+  if (templateParam) {
+    params.search.template = templateParam;
+  }
+  params.hash = Object.assign({}, params.hash, queryString.parse(location.hash.substring(1)));
+  return params;
+}
+
+// Define our iframe manager, an our css transplier
+let iframeManager = false;
```

Comment:
I guess they can be declared where needed , it is confusing when you declare them here and instantiate somewhere else.